### ISSUE TITLE
Charts, 

### ISSUE THREAD

**Issue by Knittybird** [Knittybird/Cryptogether#37]
In small screens don't display y-axis labels on at least the candlestick chart

**Comment by danieldupriest** [comment]
I don't think this is possible with css is it? It would require react to change the candlestick chart's options according to screen size. Doesn't sound easy.
